set.mm

```diff
@@ -98897,6 +99094,11 @@ is that it denies the existence of a set containing itself ( ~ elirrv ).
       CZODZEPBFZQCZDPBAAQAGZROSQAQASHZTIJBKLOMN $.
   $}
 
+  $( A class is not equal to any of its elements.  (Contributed by AV,
+     14-Jun-2022.) $)
+  elneq $p |- ( A e. B -> A =/= B ) $=
```

Comment:
do we have in set.mm `|- -. (A e. B /\ A = B )` ?
If not, it would be nice to add it (mabye with the variable-free version `( _E i^i _I ) = (/)`).

Reply:
Not as far as I know. I agree, it would be nice to have this theorem in set.mm, analogously to ~en2lp. Also the variable-free version looks interesting (is there any analog for ~en2lp)? I try to add both of them...

Reply:
The variable-free version of ~en2lp is ` ``' _E =/= _E ` (The membership (epsilon) relation is asymmetric.), or with the definition in Peter's mathbox: ` -. SymRel _E `. I'll add these theorems, too.

Reply:
Rather ```(`' _E i^i E ) = (/) ```

Reply:
Done.